templates/analytics/support.html

```diff
@@ -23,6 +23,11 @@
         <center>
             {{ message }}
         </center>
+
+        {% if "Subdomain" in message %}
+        {% set redirect_to = message.split()[-1]  %}
+        <meta http-equiv="refresh" content="1; url=/activity/support?q={{ redirect_to }}">
```

Comment:
hmm. Checking for "Subdomain" in message is not nice. You can probably set context["redirect_to"] for this case and use that for doing this check. And we should only redirect after 3 or more seconds since the success message("Subdomain changed from A -> B") should be visible to the user.

Another possible way to do this would be to make this redirect from the view instead. Anyway, do whatever is easier as long as the message is visible to the user for at least 3 seconds.

Reply:
Yes! made that change and have posted the screenshot in PR's discription..